### PR TITLE
Add celery and gravity files to .gitignore

### PR DESCRIPTION
**Pull request by davebx**
## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,10 @@
 .hg*
+
+# Celery/Gravity state files
+
+celerybeat-schedule.*
+database/gravity
```
Comment (nsoranzo) [inline .gitignore]: Should we just replace the over 20 `database/something` entries with just `database/*` ?

Comment (jdavcs) [reply .gitignore]: Maybe just add (exclude) a README with a comment (like the current info.txt) and ignore the rest?

Comment (dannon) [reply .gitignore]: IMO just ignore the whole thing.  If someone needs to change the info or add more tracked files, that's always still possible.

Comment (dannon) [reply .gitignore]: I pushed a commit doing this; hope that's fine @davebx

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**